csp-mixin/signals/gcpceOverview.libsonnet

```diff
@@ -0,0 +1,279 @@
+            legendCustomTemplate: '',
+          },
+        },
+      },
+      top5CpuUtilization: {
+        name: 'Top 5 Instances by CPU Utilitization',
+        description: 'Fractional utilization of allocated CPU on an instance',
+        type: 'gauge',
+        aggFunction: 'sum',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+            exprWrappers: [['topk(5, 100*', ')']],
+          },
+        },
+      },
+
+      top5SystemProblem: {
+        name: 'Top 5 Instances by System problem',
+        description: 'List of top 5 instances with system problems.',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'topk(5, sum by (instance_name, job, project_id) (increase(stackdriver_gce_instance_compute_googleapis_com_guest_system_problem_count{%(queriesSelector)s}[$__range])))',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+
+      top5DiskWrite: {
+        name: 'Top 5 Instances by Disk write bytes',
+        description: 'List of top 5 instances by disk write bytes',
+        type: 'raw',
+        unit: 'bytes',
+        sources: {
+          stackdriver: {
+            expr: 'topk(5, sum by (instance_name, job, project_id) (increase(stackdriver_gce_instance_compute_googleapis_com_instance_disk_write_bytes_count{%(queriesSelector)s}[$__range])))',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+
+      top5DiskRead: {
+        name: 'Top 5 Instances by Disk read bytes',
+        description: 'List of top 5 instances by disk read bytes',
+        type: 'raw',
+        unit: 'bytes',
+        sources: {
+          stackdriver: {
+            expr: 'topk(5, sum by (instance_name, job, project_id) (increase(stackdriver_gce_instance_compute_googleapis_com_instance_disk_read_bytes_count{%(queriesSelector)s}[$__range])))',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+
+      // Table
+      tableCpuUtilization: {
+        name: 'tableCpuUtilization',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: '100 * sum by (instance_name, job, project_id)(stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s})',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+      tableUptime: {
+        name: 'tableUptime',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(stackdriver_gce_instance_compute_googleapis_com_instance_uptime_total{%(queriesSelector)s})',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+      tableSentPackets: {
```

Comment:
I think **tableSentPackets, tableReceivedPackets, tableSentBytes, tableReceivedBytes, tableReadBytes, tableWriteBytes** don't need a **rate** function because are not CUMULATIVE metrics. You could convert it as `type=gauge` or better use directly existing signals **diskBytesWrite, diskBytesRead, networkReceived, networkSent, packetsReceived, packetsSent** converted to gauge. You don't need to define same signals twice just because are being used as table or as timeSeries.